priv/repo/migrations/20170803180536_add_notification_configuration.exs

```diff
@@ -0,0 +1,17 @@
+defmodule CoursePlanner.Repo.Migrations.AddNotificationConfiguration do
+  use Ecto.Migration
+
+  def up do
+    alter table(:users) do
+      add :notified, :date
+      add :notification_frequency_days, :integer, null: false, default: 1
```

Comment:
I am not sure about `frequency` in the name, when i see this name I think of `1 message/day`, or `2 message/day`